github3/github.py

```diff
@@ -996,7 +996,8 @@ def pubsubhubbub(self, mode, topic, callback, secret=''):
         :returns: bool
         """
         from re import match
-        m = match('https?://[\w\d\-\.\:]+/\w+/[\w\._-]+/events/\w+', topic)
+        m = match('https?://[\w\d\-\.\:]+/\w+[\w-]+\w/[\w\._-]+/events/\w+',
```

Comment:
Upon further reflection I don't think we should validate too strictly what GitHub says a valid username is today. They might change it, right?

How about `[\w-]+`?